### PR TITLE
fix: `noFileSelected()` computed property to handle `null` values and consequently adapt `emptySlots()` computed property

### PR DESCRIPTION
**Pull request by beneditomauro**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-util-vue/issues/332 |
| Dependencies | - |
| Decisions | - |
| Animated GIF | OFkaGKUZsh|

### CODE REVIEW

File: vue/mixins/upload.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const uploadMixin = {
     },
     computed: {
         noFileSelected() {
-            return this.filesData?.length === 0;
+            return this.filesData?.length === 0 || !this.filesData;
```
Comment (3rdvision) [inline vue/mixins/upload.js:39]: small simplification :smile: 

```suggestion
            return !this.filesData?.length;

```

**Review by 3rdvision — COMMENTED** [review]
just a small suggestion

**Review by 3rdvision — APPROVED** [review]

**Comment by 3rdvision** [comment]
Build is failing. I believe you need to port the following fixes:
1- https://github.com/ripe-tech/peri-templating/pull/55/commits/f7343478e1c2437429df6298ba8c32398bb200c4
2- https://github.com/ripe-tech/peri-templating/pull/55/commits/a281af2f896f2ea0e7fcce0b1cb089d92026c0df

@beneditomauro please check one by one as it might only need the first fix but not the second

**Review by joao-conde — APPROVED** [review]

**Comment by beneditomauro** [comment]
> Build is failing. I believe you need to port the following fixes: 1- [ripe-tech/peri-templating@f734347](https://github.com/ripe-tech/peri-templating/commit/f7343478e1c2437429df6298ba8c32398bb200c4) 2- [ripe-tech/peri-templating@a281af2](https://github.com/ripe-tech/peri-templating/commit/a281af2f896f2ea0e7fcce0b1cb089d92026c0df)
> 
> @beneditomauro please check one by one as it might only need the first fix but not the second

Neither worked, used this one: https://github.com/ripe-tech/peri-templating/pull/56/files, but not approved yet

**Comment by 3rdvision** [comment]
> > Build is failing. I believe you need to port the following fixes: 1- [ripe-tech/peri-templating@f734347](https://github.com/ripe-tech/peri-templating/commit/f7343478e1c2437429df6298ba8c32398bb200c4) 2- [ripe-tech/peri-templating@a281af2](https://github.com/ripe-tech/peri-templating/commit/a281af2f896f2ea0e7fcce0b1cb089d92026c0df)
> > @beneditomauro please check one by one as it might only need the first fix but not the second
> 
> Neither worked, used this one: https://github.com/ripe-tech/peri-templating/pull/56/files, but not approved yet

@beneditomauro  This is not the proper fix. You should have used both 1 and 2.
Fixing the version of `stylelint-config-hive` is just mitigating the wrong versions.

**Comment by 3rdvision** [comment]
@beneditomauro Since this was already merged I opened [a PR](https://github.com/ripe-tech/ripe-components-vue/pull/619) on top of `master` with that change.
Looks like after bumping stylelint versions, it got more advanced for some CSS selectors and simplified some cases :smile:

**Pull request merged**